# CVE_2017_7921_EXP

修复了源项目的一个bug,若能正常使用使用源项目即可

| Infos    | Value                                                              |
| -------- | -------------------------------------------------------------------|
| Username | [https://github.com/D2550](https://github.com/D2550) |
| Url      | [https://github.com/D2550/CVE_2017_7921_EXP](https://github.com/D2550/CVE_2017_7921_EXP)                                               |
| Stars    | 1                                                          |
| License  | Apache License 2.0                                                        |

<details>

<summary>Topic / Tags</summary>



</details>

## Readme

# CVE_2017_7921_EXP
源项目地址：https://github.com/JrDw0/CVE-2017-7921-EXP
删除了几行导致bug的代码，若能正常使用请使用源项目
# 使用
```
pip3 install requirements.txt
python3 .\CVE_2017_7921_EXP.py --target=1.txt run
```
Python3.9 Windows 11 测试通过



## Get the content

```
git clone https://github.com/D2550/CVE_2017_7921_EXP
cd CVE_2017_7921_EXP
```

{% embed url="https://github.com/D2550/CVE_2017_7921_EXP" %}

<figure><img src="https://avatars.githubusercontent.com/u/59276674?v=4" alt=""><figcaption><p>D2550</p></figcaption></figure>
